themes/cardinal_service/pdb_components/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "pdb_components",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "dev": "webpack --mode development",
+    "build": "webpack --mode production"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
```

Comment:
What is ISC?

Reply:
:shrug: Thats just what happens when i did `npm init -y`